config/configmapprovider/provider.go

```diff
@@ -12,10 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package config // import "go.opentelemetry.io/collector/config"
+package configmapprovider // import "go.opentelemetry.io/collector/config/configmapprovider"
 
 import (
 	"context"
+
+	"go.opentelemetry.io/collector/config"
 )
 
 // MapProvider is an interface that helps providing configuration's parser.
```

Comment:
I think just "Provider" is good now with the new package.

Reply:
Done.